api/URLSearchParams.json

```diff
@@ -238,10 +238,12 @@
               "version_added": "36"
             },
             "safari": {
-              "version_added": null
+              "version_added": true,
+              "partial_implementation": true
```

Comment:
Everything else here seems fine to me, but the partial implementation flags need some elaboration. Would you mind adding `"notes"` here and the entry below explaining? Thank you!

Reply:
@ddbeck Shall I link to [the test page](https://w3c-test.org/url/urlsearchparams-delete.any.html) or [the source code for failing test](https://github.com/web-platform-tests/wpt/blob/master/url/urlsearchparams-delete.any.js#L40-L45) or neither?

Reply:
Linking to the bug sounds fine to me. How about a note like `"Removing a non-existent query parameter doesn't remove <code>?</code> from the URL. See <a href=''>bug 123</a>."` Or whatever makes sense here. I'm not expert on this feature.